p8_integration_tests/scripts_test/script_checker.py

```diff
@@ -17,8 +17,8 @@
 import os
 import time
 import matplotlib
-matplotlib.use('Agg')
 import matplotlib.pyplot as plt  # noqa: E401
+matplotlib.use('Agg')
```

Comment:
Technically, you shouldn't do this.  You need to tell matplotlib what to use before you import other things as I understand it.  It seems to work here, but I wouldn't want to guarantee anything.

Reply:
pass. pep8 said it did not want a global import below a none import thing. so pass.